roles/openshift_node_bootstrap_configmap/templates/node-config.yaml.j2

```diff
@@ -0,0 +1,53 @@
+allowDisabledDocker: false
+apiVersion: v1
+authConfig:
+  authenticationCacheSize: 1000
+  authenticationCacheTTL: 5m
+  authorizationCacheSize: 1000
+  authorizationCacheTTL: 5m
+dnsBindAddress: "127.0.0.1:53"
+dnsDomain: cluster.local
+dnsIP: 0.0.0.0
```

Comment:
@smarterclayton, @michaelgugino, I moved the generation from a dynamic call to a template.  This was a change that went in https://github.com/openshift/openshift-ansible/pull/5924/files so I am moving it to the template.

Reply:
This still supports post processing for which is the happy median until we reach some understanding of config template or template + yedit.